src/pages/User-Infor/index.vue

```diff
@@ -0,0 +1,316 @@
+        required: true,
+        validator: (_, repeatPassword) => {
+            if (repeatPassword !== account.password) {
+                return new Error('Mật khẩu không giống với mật khẩu đã nhập lại!');
+            }
+            return true;
+        },
+        trigger: ['blur', 'password-input']
+    }
+}
+const { coords } = useGeolocation();
+const handleUpdateUserAddress = async () => {
+    try {
+        loadingBar.start();
+        disabledRef.value = false;
+        const params = {
+            customerId: customerId.value,
+            latitude: coords.value.latitude,
+            longitude: coords.value.longitude
+        }
+        await updateUserAddress(params);
+        loadingBar.finish();
+        disabledRef.value = true;
+        message.success('Cập nhập địa chỉ thành công!');
+        await loadCustomer();
+    } catch (err) {
+        console.log(err);
+        if (!!err.response) {
+            message.error(err.response.data.message);
+        } else {
+            message.error(err.message);
+        }
+    }
+}
+const handleUpdateUserInformation = async () => {
+    formRef.value?.validate(async (errors) => {
+        if (!errors) {
+            try {
+                loadingBar.start();
+                disabledRef.value = false;
+                let day = new Date(userForm.value.dob).getDate();
+                let month = new Date(userForm.value.dob).getMonth() + 1;
+                let year = new Date(userForm.value.dob).getFullYear();
+                console.log(userForm.value.dob);
+                let dobUser = year + '-' + month + '-' + day;
+                const newUserData = {
+                    customerId: customerId.value,
+                    fullName: userForm.value.fullName,
+                    dob: dobUser,
+                    phoneNumber: userForm.value.phoneNumber
+                }
+                const res = await updateUserInformation(newUserData);
+                console.log(res);
+                loadingBar.finish();
+                disabledRef.value = true;
+                message.success('Cập nhập thông tin thành công!');
+                loadCustomer();
+            } catch (err) {
+                console.log(err);
+                if (!!err.response) {
+                    message.error(err.response.data.message);
+                } else {
+                    message.error(err.message);
+                }
+            }
+        }
+    })
+}
+const handleChangePassword = async () => {
+    formRefAccount.value?.validate(async (errors) => {
+        if (!errors) {
+            try {
+                loadingBar.start();
+                disabledRef.value = false;
+                const newPasswordData = {
+                    oldPassword: account.oldPassword,
+                    password: account.password,
+                    repeatPassword: account.repeatPassword
+                }
+                const res = await changeUserPassword(newPasswordData);
```

Comment:
```suggestion
                const res = await changeUserPassword(account);
```
Không cần phải tạo obj mới đâu=))